poc/tests/test_vidpf.py

```diff
@@ -152,6 +149,37 @@ def test(self):
             [Field128(0), Field128(0)],
         ])
 
+    def test_exhuastive(self):
```

Comment:
Could you add a comment here on what the test does so it's easier for people to read it? It should be clear already but more comments would be good.

Reply:
Done.